client/a8c-for-agencies/style.scss

```diff
@@ -642,3 +642,7 @@ html {
 		}
 	}
 }
+
+.opacity-50 {
```

Comment:
I suggest prefixing `.is-*` here if this is a modifier to follow Calypso CSS guideline.

Reply:
Fixed it here: https://github.com/Automattic/wp-calypso/pull/95548